app/views/idv/usps/_address_on_file.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= button_to @presenter.button, idv_usps_path, method: 'put',
+  class: 'btn btn-primary btn-wide', form_class: 'inline-block mr2' %>
```

Comment:
personal style: I would indent the `class` past the `button_to` and maybe line it up with `@presenter`